public-holiday-app/pages/api/holiday.js

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  if (req.method === "GET") {
+    if (req.query.year) {
```

Comment:
You can also do something like this:

```js
const year = req.query.year || 2021;
```

If you do this, you won't need an `if...else` condition.